examples/lodepng.c

```diff
@@ -0,0 +1,6464 @@
+/*
```

Comment:
uuh, I'm not fully comfortable including this in the source.
Could we do something else instead? Like, link to `stb`, or hand-write the BMP output, or something like that (not involving a lot of code)?

Reply:
I just came across [this post](https://ibob.github.io/blog/2020/01/13/cmake-package-management/), is the current solution acceptable?

Reply:
I think I have an idea of how to make this work without resorting to C libraries laying around. We could have another mini-crate as a part of this repository that's only job is to expose this:
```rust
extern "C" fn save_png(const char* path, const char* data);
```
The crate would be a single Rust file, non-publishable, and compiling to a static lib. It would just link to `png` crate and expose this FFI function. The capture example would then link with it and use the function.

Reply:
Ok so I've grabbed the code from the capture example in [wgpu-rs](https://github.com/gfx-rs/wgpu-rs/blob/master/examples/capture/main.rs#L143) and dropped it into an example helper library.